### PR TITLE
Issue7 contact

### PR DESCRIPTION
**Pull request by sareenasaji**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,46 +1,122 @@
...
+            <div class="flex justify-between flex-col md:flex-row gap-2">
+                <label for="" class="text-xl px-1 my-1 text-center">Email</label>
+                <input type="email" name="Email" class="text-sm w-40 h-10 text-blue-800 bg-cyan-200 opacity-60 ml-1 rounded-md md:w-64 md:text-xl focus:bg-cyan-400 focus:border-2 border-white" required placeholder="ex-abc@gmail.com">
+            </div>
+            <div class="flex justify-start align-start md:justify-between flex-col md:flex-row gap-2">
+                <label for="" class="text-xl px-1 my-1 text-center">Message</label>
+                <textarea name="Message" id="" class="text-sm text-blue-800 w-40 h-16 bg-cyan-200 opacity-60 ml-1 rounded-md md:w-64 md:text-xl focus:bg-cyan-400 focus:border-2 border-white" placeholder="Enter Message"></textarea>
+            </div>
+            <button name="submit" class="bg-cyan-500 px-8 py-2 rounded-lg text-sm hover:bg-cyan-800 ease-in duration-300 hover:shadow-xl hover:text-red-400 hover:scale-110">Submit</button>
+            <span id="msg" class="bg-cyan-800 text-xl text-green-400 px-1 py-2 text-center animate-bounce font-semibold rounded-md hidden"></span>
```
Comment (kaarthikrajeev) [inline public/index.html]: Submit button is kinda hard to see.

**Review by kaarthikrajeev — APPROVED** [review]
form looks a bit stretched,adding scroll-smooth class in html would be nice otherwise LGTM!

**Review by Adhithyarajan — COMMENTED** [review]
@sareenasaji Submit button is too small. please add a border or something

**Review by Adhithyarajan — COMMENTED** [review]
Also you can try decreasing the gap between the fields in the form or placing it to the center.

**Review by Adhithyarajan — COMMENTED** [review]
@sareenasaji if you change the color of icons or if you can change the color of icons on hover, then it will be more visible

**Comment by Gowrisankar1** [comment]
Give a boxy design to the contact page.

**Pull request merged**